test/lib/usd/translators/testUsdExportCustomConverter.template.py

```diff
@@ -32,7 +32,7 @@ class testUsdExportCustomConverter(unittest.TestCase):
     def setUpClass(cls):
         # In case tests are run in parallel, make sure we export to separate
         # directories:
-        plugin_path = os.getenv("PXR_PLUGINPATH_NAME")
+        plugin_path = os.getenv("@PXR_OVERRIDE_PLUGINPATH_NAME@")
```

Comment:
Thats a neat trick using the configure_file

Reply:
Honestly, I wanted to find a way that DIDN'T have to use `configure_file`... but I couldn't think of any alternative.  At least this works!